newZealand.html

```diff
@@ -41,7 +40,7 @@ <h2>People</h2>
                 <li>NEIL FINN – TE AWAMUTU</li>
                 <li>STEVEN ADAMS – ROTORUA</li>
                 <li>FLIGHT OF THE CONCHORDS – WELLINGTON</li>
-                </ul>
+            </ul>
 
 
         </div>
```

Comment:
needs <ul> on next line, to complete the "unordered list" thought, before continuing.